openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.tsx

```diff
@@ -252,7 +252,8 @@ export const CustomPropertyTable = <T extends ExtentionEntitiesKeys>({
             <Typography.Text className="right-panel-label">
               {t('label.custom-property-plural')}
             </Typography.Text>
-            {(entityTypeDetail.customProperties ?? []).length >= maxDataCap &&
+            {maxDataCap &&
+              (entityTypeDetail.customProperties ?? []).length >= maxDataCap &&
               entityDetails.fullyQualifiedName && (
```

Comment:
Can we bring this condition out of the return statement and make it more readable?